src/System.Private.CoreLib/shared/System/Exception.cs

```diff
@@ -119,27 +120,52 @@ public virtual void GetObjectData(SerializationInfo info, StreamingContext conte
 
         public override string ToString()
         {
-            string s = GetClassName();
+            // Get the lengths of the StackTrace and _innerException first so we know how large to make the ValueStringBuilder
+            string? stackTrace = StackTrace;
+            string? innerException = null;
+            if (_innerException != null)
+            {
+                innerException = _innerException.ToString();
+            }
+
+            int initialLength = 128;
```

Comment:
Where does this `128` come from?  Would be worth a comment, even if it's just to say it's an arbitrary guess at the size required.

Reply:
Might be clearer to just have `+ 128` when calling the constructor.